voters.html

```diff
@@ -1,27 +1,30 @@
 <!DOCTYPE html>
 
 <html>
-
-<head>
-  <title>Searchy</title>
+  <head>
+  <title>Make your own mailer!</title>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" type="text/css" href="searchy.css">
-  <script type="text/javascript" src="script.js"></script>
 </head>
 
 <body>
 
-
-  <div class="title bar">
-    <h1>Searchy</h1>
+  
+  <div class="titlebar">
+    <h1>Make your own mailer!</h1>
   </div>
-  <header>
+  <div class="unitelogo">
+
+  <img src="https://raw.githubusercontent.com/morganwk/bernard/master/unite%20dark%20background%20logo.svg">
```

Comment:
I think it would be best to reference a file on our server instead of Github's. I'll figure out how to get this working later though.